src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -329,7 +329,23 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 		}, [this.listRenderer],
 			{
 				keyboardSupport: false,
-
+				accessibilityProvider: {
+					getAriaLabel: element => {
+						if (element.action instanceof CodeActionAction) {
+							const label = element.action.label;
+							if (!element.action.enabled) {
+								if (element.action instanceof CodeActionAction) {
+									localize({ key: 'customCodeActionWidget.labels', comment: ['Code action labels for accessibility.'] }, "{0}, Disabled Reason: {1}", label, element.action.action.disabled);
```

Comment:
Ah sorry I shouldn't have approved this. You still need to assign to `label` here. The call to `localize` does not have an effect otherwise

Reply:
Also for the comment, you typically also want to explain what the placeholders mean since this will not be obvious to translators

Reply:
Woops did not catch that - will add this in with the big PR with the added action bar!